package.json

```diff
@@ -4,6 +4,10 @@
   "description": "The new home of the betanextbest.wellcomecollection.org website.",
   "main": "server/app.js",
   "scripts": {
+    "lint:scss": "cd client && gulp scss:lint",
```

Comment:
Hmm. Not sure why we need to abstract this into the root package if it only relates to the client...
We could potentially start looking at something like [`lerna`](https://github.com/lerna/lerna)?